ui/app/services/system.js

```diff
@@ -23,8 +29,72 @@ export default Service.extend({
     });
   }),
 
-  namespaces: computed(function() {
-    return this.get('store').findAll('namespace');
+  defaultRegion: computed(function() {
+    const token = this.get('token');
+    return PromiseObject.create({
+      promise: token
+        .authorizedRawRequest(`/${namespace}/agent/members`)
+        .then(jsonWithDefault({}))
+        .then(json => {
+          return { region: json.ServerRegion };
+        }),
+    });
+  }),
+
+  regions: computed(function() {
+    const token = this.get('token');
+
+    return PromiseArray.create({
+      promise: token.authorizedRawRequest(`/${namespace}/regions`).then(jsonWithDefault([])),
+    });
+  }),
+
+  activeRegion: computed('regions.[]', {
+    get() {
+      const regions = this.get('regions');
+      const region = window.localStorage.nomadActiveRegion;
+
+      if (regions.includes(region)) {
+        return region;
+      }
+
+      return null;
+    },
+    set(key, value) {
+      if (value == null) {
```

Comment:
Is this loose equality intentional?

Reply:
Yeah, to be truthy for `undefined` as well.